aim/web/api/runs/audio_utils.py

```diff
@@ -0,0 +1,193 @@
+from typing import Iterable, Tuple, List
+from typing import TYPE_CHECKING
+
+from aim.storage.treeutils import encode_tree
+
+from aim.storage.context import Context
+from aim.sdk.objects import Audio
+from aim.sdk.run import Run
+from aim.sdk.sequence_collection import SequenceCollection
+from aim.sdk.sequence import Sequence
+from aim.sdk.uri_service import URIService, generate_resource_path
+
+from aim.web.api.runs.utils import get_run_props, collect_run_streamable_data, IndexRange
+from aim.web.api.runs.pydantic_models import TraceBase
+
+if TYPE_CHECKING:
+    from aim.sdk import Repo
+
+
+def sliced_audio_record(values: Iterable[Audio], _slice: slice) -> Iterable[Audio]:
+    yield from zip(range(_slice.start, _slice.stop, _slice.step), values[_slice])
+
+
+def audio_record_to_encodable(obj, trace, step, index=0):
+    resource_path = generate_resource_path(trace.values.tree.container, (step, 'data'))
+    return [{
+        'index': index,
+        'format': obj.storage.get('format'),
+        'size': obj.storage.get('size'),
```

Comment:
wrote a comment above. do we need `'size'`?

Reply:
Nice addition but completely optional